package.json

```diff
@@ -51,13 +59,20 @@
 		"chai": "4.1.2",
 		"cucumber": "4.2.1",
 		"mocha": "5.2.0",
-		"nock": "9.4.1",
 		"node-sass": "4.9.0",
 		"nyc": "12.0.2",
 		"sinon": "6.1.2",
 		"sinon-chai": "3.2.0",
 		"sinon-express-mock": "2.0.4",
+		"tough-cookie": "^2.4.3",
 		"ts-node": "7.0.0",
-		"typescript": "2.9.2"
+		"typescript": "2.9.2",
+		"@types/app-root-path": "^1.2.4",
+		"@types/chai-as-promised": "^7.1.0",
+		"chai-as-promised": "^7.1.1",
+		"daemonize-process": "^1.0.9",
+		"minimist": "^1.2.0",
+		"mockserver-client": "^5.3.0",
+		"mockserver-node": "^5.3.0"
```

Comment:
You can use the `-E` switch to install the exact version, e.g. `npm i -P -E moment`